### DISCUSSION TITLE
Statamic disable PHP front-end & use Statamic as headless only?

### DISCUSSION THREAD

**Discussion by Galanthus** [statamic/cms#9775] [General]
Hi,

I would like to integrate Nuxt JS to fetch and build my website using Statamic as the back-end and Nuxt for the front-end. Any easy way to do this?

**Comment by ryanmitchell** [comment]
I would personally run Statamic off a subdomain in this instance (eg cp.yoursite.com) and redirect any urls that dont start in /cp to /cp
